cloud/data_source_cloud_environment.go

```diff
@@ -40,16 +40,16 @@ func dataSourceCloudEnvironment() *schema.Resource {
 			},
 		},
 		Schema: map[string]*schema.Schema{
-			"organization": {
+			"name": {
 				Type:         schema.TypeString,
 				Required:     true,
```

Comment:
name is optional? @mitch-hamm

Reply:
Yes, to ensure uniqueness in the cluster we are generating the name based on the other params plus a random hash. If you are admin (streamnative) you can supply a name but otherwise it is generated

Reply:
so I think need update `Required:    false`

Reply:
This is in the data source, so it does need to be there to read. In the resource it's removed since customers won't be able to use it at all

Reply:
see, thanks for explain